widgetssdk/src/main/res/values/styles.xml

```diff
@@ -75,6 +75,12 @@
         <item name="operatorPlaceholderIconTintList">?attr/baseLightColor</item>
     </style>
 
+    <style name="Application.Glia.VisitorCode" parent="TextAppearance.MaterialComponents.Headline4">
+        <item name="android:textStyle">bold</item>
+        <item name="android:textColor">?attr/gliaBaseDarkColor</item>
+        <item name="android:background">@drawable/bg_char_slot</item>
```

Comment:
If this is TextAppearance it won't have a background. Anyway, I'd suggest using this background directly inside our xml, or not using anymore because I saw you implemented this in View class.